browser/brave_profile_prefs.cc

```diff
@@ -193,6 +193,9 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
   // Autocomplete in address bar
   registry->RegisterBooleanPref(kAutocompleteEnabled, true);
 
+// Restore "Other Bookmarks" migration
```

Comment:
nit: seems missing space before the comment

Reply:
fixed